src/Pages/Platforms/platformTabs.tsx

```diff
@@ -58,7 +60,9 @@ export const PlatformTabs: React.SFC<PlatformTabsProps> = ({ match }) => {
 
           {/* Display our pages for the platform. */}
           <Switch>
-            <Route path={paths.platforms.all}>{(props) => <ErddapAllObservationsTable {...platform_props} />}</Route>
+            <Route path={paths.platforms.all}>
+              {(props) => <ErddapAllObservationsTable {...platform_props} unit_system={unit_system} />}
```

Comment:
Codacy found an issue: [Identifier 'platform_props' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)